sci-biology/squizz/squizz-0.99b.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+DESCRIPTION="squizz is a sequence format file checker/converter."
```

Comment:
You can drop $PN from DESCRIPTION

Something like

DESCRIPTION="Sequence format file checker/converter"

would be absolutely enough